setup.py

```diff
@@ -91,7 +91,6 @@
         "pandas>=1.3.0,<2",
         "pluggy>=1.0.0,<2",
         "pydantic>=1.9.2,<2",
-        "pygit2>=1.7.2,<2",
         "PyGithub>=1.54,<2",
```

Comment:
this doesn't depend on pygit2 as well?

Reply:
This PR replaces the usage of pygit2 with `subprocess.run(["path/to/git"])`.
What do you mean by `as well`?

Reply:
@unparalleled-js I was wondering if PyGithub had pygit2 as a dep as well but if `pip freeze | grep pygit2` was empty. This is sorted out! :+1: